internal/util/util.go

```diff
@@ -32,6 +30,9 @@ const GCSFUSE_PARENT_PROCESS_DIR = "gcsfuse-parent-process-dir"
 const Sequential = "Sequential"
 const Random = "Random"
 
+const MaxMiBsInUint64 uint64 = math.MaxUint64 >> 20
```

Comment:
We can consistent with MiB or MB.

I feel, in our code base we have used MB (in place of MiB).

Reply:
I've made it MiBs everywhere in my change for now as that I had used that most places earlier. I've chosen MiBs as it's more explicit and less-confusing.